install-dev/upgrade/sql/1.7.2.2.sql

```diff
@@ -0,0 +1,3 @@
+SET NAMES 'utf8';
+
+ALTER TABLE `PREFIX_customization_field` ADD `is_deleted` TINYINT(1) NOT NULL DEFAULT '0';
```

Comment:
If this is for the 1.7.2.3 release, you have to change the file name or this file won't be applied.

Reply:
Too late, let's rebase it on develop for 1.7.3

Reply:
Same, if it is for 1.7.3, it needs to be done on tve 1.7.3.0.sql !  Don't forget

Reply:
Needs to be made on 1.7.3.0.sql